core/docz-components/src/theme/index.ts

```diff
@@ -86,7 +89,7 @@ const getTheme = (...args) =>
         py: 3,
         px: 4,
         bg: 'blockquote.bg',
-        borderLeft: t => `5px solid ${t.colors.blockquote.boder}`,
+        borderLeft: (t: Theme) => `5px solid ${t.colors.blockquote.boder}`,
```

Comment:
Typo in `t.colors.blockquote.boder`


`boder` -> `border`

Reply:
Thanks for catching that 👍